assets/tests/components/ladderPage.test.tsx

```diff
@@ -480,14 +480,14 @@ describe("LadderPage", () => {
       ...initialState,
       selectedNotification: notification,
     }
-    const wrapper = mount(
+    const result = render(
       <StateDispatchProvider state={state} dispatch={jest.fn()}>
         <BrowserRouter>
           <LadderPage />
         </BrowserRouter>
       </StateDispatchProvider>
     )
-    expect(wrapper.find("#m-properties-panel").exists()).toBeFalsy()
+    expect(result.queryByText(/vehicles/)).toBeNull()
```

Comment:
If I'm understanding this correctly (I assume it's looking for the text in the title bar?), I think the v should either be capitalized or the regex should be made case-insensitive.